ui/core/src/store/index.ts

```diff
@@ -1,14 +1,16 @@
-import { reactive, Ref, toRefs } from "@vue/reactivity";
+import { reactive } from "@vue/reactivity";
 import { wallet, WalletStore } from "./wallet";
 import { asset, AssetStore } from "./asset";
 import { pools, PoolStore } from "./pools";
-import { notifications, NotificationsStore } from "./notifications"
+import { notifications, NotificationsStore } from "./notifications";
+import { LiquidityProvider, Pool } from "../entities";
 export * from "./poolFinder";
 
 export type Store = {
   wallet: WalletStore;
   asset: AssetStore;
   pools: PoolStore;
+  accountpools: { lp: LiquidityProvider; pool: Pool }[];
```

Comment:
What's this

Reply:
A bad name for a thing. Basically I need a LiquidityProvider object and a Pool object as they are both related in a 1:1 relationship. I would prefer to refactor core get the names right and do this in a neater way but I have been in a rush.

Reply:
LIquidityProvider has the user's poolUnits on it while the pool has the total units on it

Reply:
could possibly be `mypools`?

Reply:
but then they are not "pools"